radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/particles/quarks/UniqueQuark.java

```diff
@@ -0,0 +1,20 @@
+package com.radixdlt.client.core.atoms.particles.quarks;
+
+/**
+ * Specifies a uniqueness constraint on an atom. That is, only one atom can
+ * contain this particle defined by it's byte array contents and it's owner.
```

Comment:
Quarks should have the constraint that they don't know anything about any other quarks. So UniqueQuark should not know about an existing OwnableQuark. Instead, uniqueness here is determined by:
* byte array
* encapsulating particle class
* a shardable (addresses only in current version)